github/ci/prow-deploy/kustom/base/manifests/local/referee-deployment.yaml

```diff
@@ -21,6 +21,7 @@ spec:
         image: quay.io/kubevirtci/referee:v20250121-1e74906d2
         args:
         - --dry-run=false
+        # - --max-no-of-allowed-retest-comments=3 TODO enable this after image update
```

Comment:
Thanks for reining in our retry bot. It causes us to ignore bad code and bad infrastructure. Currently, we accept tests that pass only 20% of the time. 

However, I don't understand the motivation of this PR's complexity. Why is it better to hard-code the number 3 here, rather than [there](https://github.com/kubevirt/project-infra/pull/4019/files#diff-a45b302c808e9440bfc137b20de31f305a45205e37a7ecf926a1ad5fb110bb29R38) ?

Reply:
if we enable this it will cause the deployment of the referee to fail, since it doesn't know (yet) about the flag - therefore we are going to enable it in another (one line) PR after the deployment.

Reply:
because we can then changed the value in the deployment, i.e. we don't have to create a new image, redeploy everything